R/SS_RunJitter.R

```diff
@@ -26,7 +20,13 @@
 #' @param init_values_src Either zero or one, specifying if the initial values to
 #'   jitter should be read from the control file or from the par file, respectively.
 #'   The default is `NULL`, which will leave the starter file unchanged.
+#' @template exe
+#' @template verbose
+#' @param ... Additional arguments passed to r4ss::run(), such as
```

Comment:
Wrap function name
```
[r4ss::run()]
```
and if R CMD check won't let you use the :: for a function within the package that you are building itself, then `r4ss::run()`.

Reply:
This comment carries forward to additional uses of `...`

Reply:
Done throughout. 
I was happy to recently discover shift-ctl-H brings up global find-replace in vscode.

Reply:
Follow-up: the links between help pages created by `[r4ss::run()]` worked just fine in the installed package.